src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -67,6 +68,8 @@ public void storeMessage(Message message) {
         UUID id = UUID.fromString(idString);
         String text = (String) entity.getProperty("text");
         long timestamp = (long) entity.getProperty("timestamp");
+        String recipient = (String) entity.getProperty("recipient");
+        Message message = new Message(id, user, text, timestamp, recipient);
```

Comment:
The massage variable declared here is being rewritten on the next line. Which of them should be kept?

Reply:
The first one, with the recipient parameter - forgot to take that second one out.

Reply:
How do i make that change?

Reply:
I don't personally now, but it must be possible. I suggest you try searching it online or in the shared resources on the codeu website. Lauren may also know given that she has more experience with GitHub. But if you get stuck reach out to me again and I'll find some time to investigate it.

Reply:
Figured it out!